guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb

```diff
@@ -0,0 +1,1555 @@
+{
```

Comment:
<p>Add a Note here using the same cell style as in the note at the very bottom:</p><p> </p><p>"</p><p>Note: Spatial Data Engineering using SeDF builds on top of core Data Engineering concepts in Python. If you are new to Pandas, NumPy and related libraries, we recommend you start with the <a href="part1-introduction-to-dataengineering/" target="_blank">Introduction to Data Engineering</a> guide series and then come here.</p><p>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>